php/class-widget-posts.php

```diff
@@ -177,14 +183,27 @@ function register_instance_post_type() {
 
 		$args = array(
 			'labels' => $labels,
-			'public' => false,
+			'description' => __( 'Widget Instances.', 'mandatory-widgets' ),
+			'public' => true,
 			'capability_type' => static::INSTANCE_POST_TYPE,
+			'capabilities' => array(
+				'create_posts' => 'do_not_allow',
+			),
```

Comment:
@davidlonjon I'm not seeing the “Widget Instances” appear at all when `capability_type` is set to this here. Did you test as a superadmin?

You may need to manually map the capabilities that it generates for you, as in Widget Favorites:
https://github.com/xwp/wp-widget-favorites/blob/5758f5d7ea3c0c90c9923c3056cea8300eec3a1f/php/class-plugin.php#L100-L109